injected/integration-test/page-objects/results-collector.js

```diff
@@ -0,0 +1,200 @@
+import { readFileSync } from 'fs';
+import {
+    mockAndroidMessaging,
+    mockWebkitMessaging,
+    mockWindowsMessaging,
+    simulateSubscriptionMessage,
+    wrapWebkitScripts,
+    wrapWindowsScripts,
+} from '@duckduckgo/messaging/lib/test-utils.mjs';
+import { perPlatform } from '../type-helpers.mjs';
+
+/**
+ * This is designed to allow you to execute Playwright tests using the various
```

Comment:
Why do we keep copying this boilerplate over and over? I get that there are subtle differences but overall this is a lot of code for how small the actual test is.

Can we file an issue to clean this up at the minimum?

Reply:
Like the original point was pages.spec.js should handle all the tests and instead we've copied that into this file, web-compat and DBP etc.

It's 200 loc boilerplate we've copied per ~100 loc test code.